src/sentry/static/sentry/app/components/avatar/baseAvatar.jsx

```diff
@@ -1,16 +1,18 @@
-import MD5 from 'crypto-js/md5';
 import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
 import qs from 'query-string';
 import styled from 'react-emotion';
 
-import ConfigStore from 'app/stores/configStore';
 import LetterAvatar from 'app/components/letterAvatar';
 import Tooltip from 'app/components/tooltip';
 
+import {imageStyle} from './styles';
+import Gravatar from './gravatar';
+
 const DEFAULT_GRAVATAR_SIZE = 64;
 const ALLOWED_SIZES = [20, 32, 36, 48, 52, 64, 80, 96, 120];
+const DEFAULT_REMOTE_SIZE = 120;
```

Comment:
Always fetch the same size regardless of actual size in UI, otherwise the smaller sized avatars do not look good.